symengine/expression.h

```diff
@@ -32,16 +32,16 @@ class Expression
     //! Construct Expression from `int`
     Expression(int n) : m_basic(integer(n)) {}
     //! Construct Expression from Basic
-    template <typename T, typename = typename std::enable_if<std::is_constructible<RCP<const Basic>, T &&>::value>::type>
+    template <typename T>
```

Comment:
@certik yes I understand. I hope ICC and MSVC will keep on progressing in their support for C++11, the more compiler/platform coverage the better.

For what it's worth, I just tested the ``integer``, ``rational`` and ``real`` classes and they still compile with ICC.